dotcom-rendering/docs/contributing/where-should-my-code-live.md

```diff
@@ -4,75 +4,100 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
```

Comment:
I rewrote this documetation - you can see the new file here if github is lagging too much for you: https://github.com/guardian/dotcom-rendering/blob/6544a4c7a2e7a634234391d887c1296332ee1927/dotcom-rendering/docs/contributing/where-should-my-code-live.md